src/components/RightSidebar/SharedItems/SharedItemsTab.vue

```diff
@@ -223,7 +224,13 @@ export default {
 	}
 }
 
-.tab-loading {
-	margin-top: 50%;
+.shared-items {
+	&__loading {
+		margin-top: 50%;
+	}
+
+	&__empty-content {
+		margin-top: 20vh;
+	}
```

Comment:
Same for the breaking rooms tab

Reply:
Default centering (let's call it geometrical 50%) seemed a bit low to me.It will be visible very well with both EmptyContent in chat and sidebar (will provide a screenshots asap).
What's why I'd like to compensate it by lifting up a bit

Reply:
`20vh` always has a different position because it depends on window height but doesn't consider element size. So the position is a bit random: on some screens it's on the top, on others close to the center or centered, on small has a scroll bar for the margin.

4K + 150% / very up | 2K | 1K / aka centered | 1K + scale (for a11y) / scrolled
---|---|---|---
![image](https://github.com/nextcloud/spreed/assets/25978914/bf5a1db8-1ab2-494c-b8f5-b0f90e7234d7) | ![image](https://github.com/nextcloud/spreed/assets/25978914/1c57411c-3cab-42dc-b49b-95be1c29ca5f) | ![image](https://github.com/nextcloud/spreed/assets/25978914/79432056-ed03-41bc-921a-13e55a8391f6) | ![image](https://github.com/nextcloud/spreed/assets/25978914/1ab838f1-e005-4cac-9bfc-8fec8f2d6997)

Also making this behavior default was proposed by Jan and John:
https://github.com/nextcloud-libraries/nextcloud-vue/pull/4506#issuecomment-1708600189

Reply:
IMO, alternative could be a fixed margin on large screen, if we want it to be on the top

Reply:
As an alternative, I'd use some fixed margin, not dependent on the window height, if we want it to be on the top of the tab.

Reply:
> Same misposition for 50% is what confuses me as well. Take a look at difference between chat and sidebar tab below.

It is not the same, it is only about different positions between tabs and chat. But it is in the center of the container on any window. There is no scrolling for a margin on a small screen, for example.

`margin: 20vh` also has a different position comparing chat and tab, because it also doesn't count heading and buttons. It is not aligned anyway, because it is not positioned relative to a window.